R/dvhydrograph-render.R

```diff
@@ -287,6 +300,11 @@ getDVHydrographPlotConfig <- function(plotItem, plotItemName, yLabel="", minMaxE
                            lty=ifelse(plotItem[['newSet']] == "est", 1, 6), col=ifelse(plotItem[['newSet']] == "est", "orange", "red3")),
                       styles$est_lines)
     ),
+    estimated4Edges = list(
+      arrows = append(list(x0=plotItem[['time']], x1=plotItem[['time']], y0=plotItem[['y0']], y1=plotItem[['y1']], 
+                           lty=ifelse(plotItem[['newSet']] == "est", 1, 7), col=ifelse(plotItem[['newSet']] == "est", "black", "red3")),
```

Comment:
So, in styles, I have:

estimated1EdgesEst = list(lty=1, col = "blue", lwd=1, code=1, length = 0),
estimated1EdgesStat = list(lty=2, col = "red", lwd=1, code=1, length = 0),

and in render I have: 
```
estimated1Edges = list( 
      arrows = append(list(x0=plotItem[['time']], x1=plotItem[['time']], y0=plotItem[['y0']], y1=plotItem[['y1']],  
                      ifelse(plotItem[['newSet']] == "est", styles$estimated1EdgesEst, styles$estimated1EdgesStat)), styles$est_lines) 
    ),
```

but I get an error when running: 

Error in (function (x0, y0, x1 = x0, y1 = y0, length = 0.25, angle = 30,  : 
  invalid arrow head angle
 attempting to run: try({dvhydrograph(fromJSON("report-26e4c9bb-5389-4df2-9343-dd241b4cf8dd.json"), "kmschoep")},silent=TRUE)